Source/Input/ConfigManager/ConfigManager.cpp

```diff
@@ -0,0 +1,138 @@
+#include "ConfigManager.h"
+
+
+bool LogCategory::IfElementExists(const std::string& keyValue) const {
+
+	if (params.find(keyValue) != params.end()) {
+		return true;
+	}
+
+	return false;
+}
+
+std::string LogCategory::GetValue(std::string keyValue) {
+
+
+	if (IfElementExists(keyValue)) {
+
+		return params[keyValue];
+	}
+
+	std::cerr << "Key value not found" << std::endl;
+	system("PAUSE");
```

Comment:
we should NOT pause the game if some error happens! please remove any remaining pauses